data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,1660 @@
+			choice
+				`	"That low? Well, here you go mate."`
+				`	"I don't have that kind of money. Maybe we could negotiate?"`
+					goto negotiate
+				`	"There's no way I'm paying so much for an item I haven't even seen! I challenge your warlord to a fight."`
+					goto challenge
+			`	He seems disappointed, but he shows you to the item, a little cube, and you bring it back to your ship without any trouble. He also signals the local warships belonging to the warlord to let you leave without trouble.`
+			action
+				payment -10000000
+			``
+				goto leaveinpeace
+			label negotiate
+			`	"I can cut it down by... 2 million. Last chance, boy. 8 million credits, what do ya say?"`
+			choice
+				`	"Deal!"`
+				`	""There's no way I'm paying so much for an item I haven't even seen! I challenge your warlord to a fight.""`
+					goto challenge
+			`	He seems disappointed to have only 8 million credits, but you feel like he still had the upper hand in that trade. He shows you to the item, a little cube, and you bring it back to your ship without any trouble. He also signals the local warships belonging to the warlord to let you leave without trouble.`
+			action
+				payment -8000000
+			label leaveinpeace
+			action
+				set "unfettered: brokered deal"
+				"reputation: Mhairi's Legion" >?= 1
+			``
+				accept
+			label challenge
+			`	You launch your fleet and send a challenge to the warlord, putting her authority and competence in question. She cannot resist the urge to come and kill you, you only hope you can beat her and collect the item from her ship.`
+				flee
+	on accept
+		outfit "Mystery Cube"
+
+
+mission "Unfettered: Special Cargo 2"
+	landing
+	name `Mystery Gathering`
+	description `Collect a tribute of food from <stopovers>, and place the mystery cube without anyone noticing.`
+	source "Warfeed"
+	stopover "Mirrorlake"
+	cargo "food (tribute)" 93
+	blocked `You need <tons> of cargo space for the next Unfettered mission.`
+	to offer
+		has "reputation: Hai" >= 0
+		has "Unfettered: Special Cargo: done"
+	on offer
+		conversation
+			branch bought
+				has "unfettered: brokered deal"
+			branch stole
+				has "Unfettered: Special Cargo 1B: done"
+			action
+				log `Recovered a mystery cube from a pirate ship in the heat of battle.`
+			``
+				goto afterlog
+			label stole
+			action
+				log `Managed to steal a very precious mystery cube from a pirate ship.`
+			``
+				goto afterlog
+			label bought
+			action
+				log `Bought a mystery cube off pirates for the Unfettered. Price tag was large.`
+			label afterlog
+			`You find He-Yi in his war room, which is full of strange objects - only half of which appear to be weapons. You notice some keystone carvings, but also a statue, probably of some Hai goddess. He seems to have multiple missives, serving as his eyes and ears everywhere. You bring him the cube. "Very well done! But this is not all. You will have a big [reward, payment] once the mission is completed."`
```

Comment:
> He seems to have multiple missives, serving as his eyes and ears everywhere.

What do you mean?

Reply:
Well the warlord has apprentices/acolytes that tell him all thats going on

Reply:
https://www.oxfordlearnersdictionaries.com/definition/english/missive?q=missive

Reply:
Right, I'll change it to disciples